pyphare/pyphare/__main__.py

```
@@ -0,0 +1,188 @@
+#
+#
+#
+
+import os
+import yaml
+import inspect
+import importlib
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)